fonts.css

```diff
@@ -1,66 +1,172 @@
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.woff') format('woff'), /* Modern Browsers */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.ttf') format('truetype'), /* Safari, Android, iOS */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.svg#Montserrat') format('svg'); /* Legacy iOS */
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 600;
+  src: url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.eot');
+  
+  /* IE9 Compat Modes */
+  
+  src:
+          local('Montserrat SemiBold'),
+          local('Montserrat-SemiBold'),
+          url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.eot?#iefix') format('embedded-opentype'),
+    
+        /* IE6-IE8 */
+  
+         url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.woff2') format('woff2'),
+    
+       /* Super Modern Browsers */
+  
+         url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.woff') format('woff'),
+    
+        /* Modern Browsers */
+  
+         url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.ttf') format('truetype'),
+    
+        /* Safari, Android, iOS */
+  
+          url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.svg#Montserrat') format('svg');
+  
+        /* Legacy iOS */
 }
-/* montserrat-700 - latin_cyrillic-ext_cyrillic */
+
+    /* montserrat-700 - latin_cyrillic-ext_cyrillic */
+
 @font-face {
-    font-family: 'Montserrat';
-    font-style: normal;
-    font-weight: 700;
-    src: url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.eot'); /* IE9 Compat Modes */
-    src: local('Montserrat Bold'), local('Montserrat-Bold'),
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.woff2') format('woff2'), /* Super Modern Browsers */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.woff') format('woff'), /* Modern Browsers */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.ttf') format('truetype'), /* Safari, Android, iOS */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.svg#Montserrat') format('svg'); /* Legacy iOS */
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 700;
+  src: url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.eot');
+  
+  /* IE9 Compat Modes */
+  
+  src:
+          local('Montserrat Bold'),
+          local('Montserrat-Bold'),
+          url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.eot?#iefix') format('embedded-opentype'),
+    
+       /* IE6-IE8 */
+  
+         url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.woff2') format('woff2'),
+    
+        /* Super Modern Browsers */
+  
+         url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-700.woff') format('woff'),
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)